api/jobs.yaml

```diff
@@ -116,6 +131,45 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    required:
+      - required
+    properties:
+      required:
+        type: boolean
+        description: Whether or not authentication is required
+      client_id:
```

Comment:
Per doc -> mv this one back to environment.ts

Reply:
Done.